greeting-emails/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "greeting-emails",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "tsc",
+    "test": "npx knex migrate:rollback && npx knex migrate:up && jest",
+    "lint": "eslint . --ext ts",
+    "lint-fix": "eslint --fix . --ext ts"
```

Comment:
This fixes an issue where removing the DBOS eslint plugin won't work -- should likely be backported to the SDK. cc @chuck-dbos 

(we don't use the dbos linting here because it errors when we use `setTimeout`, which for the sake of this tutorial we want, for now at least, and we don't want to start adding eslint ignore lines in the tutorial...)